projects/fusion-e2e/entities/components/chip-filter/chipFilter-component.ts

```diff
@@ -0,0 +1,22 @@
+import {ChipFilterTestIdModifiers} from '@ironsource/fusion-ui/entities/test-ids-modifiers';
+import {getTestId, getTestIdSelector} from '../../global/utils';
+import {BaseComponent} from '../base-component';
+import {Locator} from '@playwright/test';
+
+export class ChipFilterComponent extends BaseComponent {
+    constructor(page, selector: string) {
```

Comment:
A note we already talked about - Chip filter should contain properties and you need to use them within your methods.
Your methods should me implemented on those properties and not like this await this.getLocator(getTestIdSelector(getTestId(testId, ChipFilterTestIdModifiers.CHIP_FILTER)));

instead set the selector for chip filter in the constructor. very important that you will implement all components like so properly!

Reply:
Done